src/app/docs/date-range-picker/date-range-picker-docs.component.html

```diff
@@ -0,0 +1,23 @@
+<sky-docs-demo-page
+  moduleName="SkyDateRangePickerModule"
+  moduleSourceCodePath="src/app/public/modules/date-range-picker/"
+  pageTitle="Date range picker"
+>
+  <sky-docs-demo-page-summary>
+    The date range picker component creates a text input with a dropdown for users to select a date range from a set of well-known options. This includes options that enable users to specify dates to create their own date range. The text input is a composite component of
+    <a href="https://developer.blackbaud.com/skyux/components/datepicker"> up to two <code>sky-datepicker</code> components</a>.
+  </sky-docs-demo-page-summary>
+
+  <!-- TODO: add error handling section once code blocks are supported. https://developer.blackbaud.com/skyux/components/date-range-picker#error-handling -->
+
+  <!-- <sky-docs-code-examples
+    [packageDependencies]="{
+      '@skyux/datetime': '*'
+    }"
+  >
+    <sky-docs-code-example
+      sourceCodePath="src/app/public/plugin-resources/code-examples/date-range-picker/basic/"
+    >
```

Comment:
Correct. I added a <!-- TODO --> line in the comments, but haven't created an issue for it since I don't know enough about the repo to determine if that functionality exists. I was going to wait on Steve and then we can add it later in a separate PR.

Reply:
I went ahead and pinged Steve since I had a list of 3 other items for him to follow up on with us when he gets back.